MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm

```diff
@@ -1007,10 +1007,12 @@
 	addVkFormatDesc( S8_UINT, Stencil8, Invalid, Invalid, Invalid, 1, 1, 1, DepthStencil );
 
 	addVkFormatDesc( D16_UNORM, Depth16Unorm, Depth32Float, Invalid, Invalid, 1, 1, 2, DepthStencil );
+#if !MVK_VISIONOS
 	addVkFormatDesc( D16_UNORM_S8_UINT, Invalid, Depth16Unorm_Stencil8, Invalid, Invalid, 1, 1, 3, DepthStencil );
 	addVkFormatDesc( D24_UNORM_S8_UINT, Depth24Unorm_Stencil8, Depth32Float_Stencil8, Invalid, Invalid, 1, 1, 4, DepthStencil );
 
-	addVkFormatDesc( X8_D24_UNORM_PACK32, Invalid, Depth24Unorm_Stencil8, Invalid, Invalid, 1, 1, 4, DepthStencil );
+    addVkFormatDesc( X8_D24_UNORM_PACK32, Invalid, Depth24Unorm_Stencil8, Invalid, Invalid, 1, 1, 4, DepthStencil );
+#endif
```

Comment:
The way we handle `MTLPixelFormats` that do not exist on some platforms, is to `#define` that `MTLPixelFormat` as `MTLPixelFormatInvalid` for those platforms in a section at the top of this file.

The mapping logic of `MTLPixelFormats` then automatically handles its absence.